app/src/components/navigation/TopBar.spec.ts

```diff
@@ -1,32 +1,41 @@
-import { describe, it, expect, vi } from "vitest";
+import "fake-indexeddb/auto";
+import { describe, it, expect, vi, beforeEach } from "vitest";
 import { RouterLinkStub, mount } from "@vue/test-utils";
 import TopBar from "./TopBar.vue";
 import * as auth0 from "@auth0/auth0-vue";
+import { createTestingPinia } from "@pinia/testing";
 import { ref } from "vue";
-import ProfileMenu from "./ProfileMenu.vue";
+import { setActivePinia } from "pinia";
 
 vi.mock("@auth0/auth0-vue");
 
-vi.mock("vue-router", () => ({
-    RouterLink: RouterLinkStub,
-    useRoute: vi.fn().mockImplementation(() => ({
-        name: "home",
-    })),
-}));
+vi.mock("vue-router", async () => {
+    const actual = await vi.importActual("vue-router");
+    return {
+        ...actual,
+        RouterLink: RouterLinkStub,
+        useRoute: vi.fn().mockImplementation(() => ({
+            name: "home",
+        })),
+    };
+});
 
 describe("TopBar", () => {
+    beforeEach(() => {
+        setActivePinia(createTestingPinia());
+    });
+
     it("shows a login button when logged out", async () => {
         (auth0 as any).useAuth0 = vi.fn().mockReturnValue({
             isAuthenticated: ref(false),
         });
 
         const wrapper = mount(TopBar);
 
-        expect(wrapper.html()).toContain("Log in");
-        expect(wrapper.findComponent(ProfileMenu).exists()).toBe(false);
+        expect(wrapper.html()).toContain("Profile");
     });
```

Comment:
Description of this test is not valid anymore. Test could be combined with the next test, as the profile menu should now always show?